tools/site-creator/create-site.js

```diff
@@ -19,6 +19,42 @@ export const SITE_CREATION_STATUS = {
   NO_FSTAB: 2,
 };
 
+function getLibraryConfigJson(org, site) {
+  const basePath = `https://content.da.live/${org}/${site}/.da/library`;
+
+  const configJson = {
+    data: {
+      total: 1,
+      limit: 1,
+      offset: 0,
+      data: [{}],
+      ':colWidths': [],
+    },
+    library: {
+      total: 2,
+      limit: 2,
```

Comment:
I think this would change too, if you delete icons.
```suggestion
      total: 1,
      limit: 1,
```